src/layers.rs

```diff
@@ -141,3 +141,60 @@ impl Module for LayerNorm {
         )
     }
 }
+
+/// Linear fully-connected layer.
+#[derive(Debug)]
+pub struct Linear {
+    ws: Tensor,
+    bs: Option<Tensor>,
+}
+
+impl Linear {
+    pub fn new<'a>(
+        vs: impl Borrow<Path<'a>>,
+        in_features: i64,
+        out_features: i64,
+        bias: bool,
+    ) -> Self {
+        let vs = vs.borrow();
+
+        let bs = if bias {
+            let bound = 1.0 / (in_features as f64).sqrt();
+            let bs_init = Init::Uniform {
+                lo: -bound,
+                up: bound,
+            };
```

Comment:
Shouldn't the bias be initialized to zero?

https://github.com/huggingface/transformers/blob/94c99db34cf9074a212c36554fb925c513d70ab1/transformers/modeling_bert.py#L487

Reply:
I guess so! I was using initialization similar to the PyTorch Linear layer, but I didn't notice that they are overriding this globally.

Reply:
This override caused some pain when I tried to initialize the bias of the softmax layer to reflect the label distribution.

Reply:
I can see that, globals are bad. I have now made the initializations arguments to `new` (since the weight initialization was also wrong) and introduced a `bert_linear` function to create a linear layer with BERT's initialization.